documentacion/posintegrado/README.md

```diff
@@ -0,0 +1,305 @@
+# POS Integrado
+
+## Cómo empezar
+
+El SDK para POS Integrado cuenta de 2 partes, una librería/SDK en C, que debe ser instalado en tu maquina para poder ser utilizado por la segunda pieza. Esta pieza es un Wrapper para accede a las funciones disponibles en la librería C.
+
+Adicionalmente, necesitaras tener instalados los drivers correspondientes a tu tarjeta de
+puerto serial o adaptador USB Serial.
+
+<aside class="notice">
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232.
+</aside>
+
+<aside class="notice">
+Tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
+</aside>
+
+<aside class="success">
+Estos drivers son conocidos por funcionar con Adaptadores genéricos que utilicen el chip CH340: http://www.wch.cn/download/CH341SER_EXE.html <br>
```

Comment:
The link should be HTML code?
<a href="http://www.wch.cn/download/CH341SER_EXE.html" target="_blank">descargar</a>
<br />

Reply:
It should be a MD link